scipy/optimize/nonlin.py

```diff
@@ -1373,6 +1373,10 @@ class KrylovJacobian(Jacobian):
         the iterative solvers in `scipy.sparse.linalg`.
 
         The default is `scipy.sparse.linalg.lgmres`.
+    inner_maxiter : int, optional
+        Parameter to pass to the \"inner\" Krylov solver: maximum number of
```

Comment:
Inside triple quote strings `"""..."""` you don't need to escape double quotes.

Reply:
I fixed it.